requirements.txt

```diff
@@ -8,3 +8,4 @@ pyrealsense2==2.41.0.2666; python_version < '3.8'
 numpy==1.19.2
 protobuf==3.13.0
 Shapely==1.7.1
+psutil==5.8.0
```

Comment:
This change is needed for unittests. Is it acceptable? I understand that in this way the requirements will grow to unwanted size.

Reply:
I would remove it :( ... note, that you can mock import so that unittest for parsing is still OK

Reply:
ok, done. Thanks for hint